.github/workflows/build.yaml

```diff
@@ -0,0 +1,101 @@
+name: build_and_test
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11", "3.12"]
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Need to clone everything for the git tags.
+          fetch-depth: 0
+
+      - uses: conda-incubator/setup-miniconda@v3
```

Comment:
It doesn't look like you need conda for this build does it? Will be faster if you don't. The pytest stuff can come from pypi. We use conda in butler so we can easily get postgres installed.

Reply:
As you might have guessed, I'm flying a little blind here and tried to use another `build.yaml`.   I'll remove that.  I'm having a bigger issue with getting `obs_subaru` installed, which is failing.  That package is used as part of the unit tests.   I'd rather not have that dependency, but I'm at a loss for what to use as a replacement to test the functionality I'm looking for.   Maybe mock-ing this is what I need to do instead.  Any thoughts?

Reply:
You can't depend on obs_subaru on GitHub. I haven't looked at your tests to see how much they really need obs_subaru vs needing a generic repository with some datasets in it.

Reply:
ok, I'll take a closer look at what I need to do.

Reply:
The dependency was removed, and the code coverage for that class is much higher.